React/Views/RCTView.m

```diff
@@ -154,6 +154,15 @@ - (NSString *)accessibilityLabel
   return RCTRecursiveAccessibilityLabel(self);
 }
 
+- (NSString *)accessibilityHint
```

Comment:
Why is this needed? Can't we just use the prop directly?

Reply:
You're right, it is not needed. I tested after removing it, and it still works. I've updated this in the latest commit, 196ce0658b7c71ec5f30b88018d784b49272cc02